src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 20;
+  const preferDiscount = 50;
+  const dayPrice = 40;
+
+  if (days < 3) {
+    return days * dayPrice;
```

Comment:
You use the expression: `(days * 40`) in other places, so let's create a separate variable for this.